### ISSUE TITLE
Automatically activate snippet not working

### ISSUE THREAD

**Issue by d2718nis** [mdelobelle/obsidian_supercharged_links#156]
# Description
Advanced option "Automatically activate snippet" has no effect until you open the "Supercharged Links" plugin settings menu.

# Steps to reproduce
1. Create a new Obsidian vault
2. Install and enable "Supercharged Links" and "Style Settings" plugins
3. Add target attributes, selector and style it in the "Style Settings" plugin as suggested in the README.md
4. Reload/restart the app

# What's the expected result?
`Options -> Appearance -> CSS snippets -> supercharged-links-gen` is enabled at the Obsidian start up when the "Automatically activate snippet" is ON.

# What's the actual result?
`Options -> Appearance -> CSS snippets -> supercharged-links-gen` is enabled only after you navigate to the "Supercharged Links" plugin settings. Therefore no styles applied until you click on the plugin settings.

# Additional details / screenshot
![screenshot-2022-11-06-01-58-37](https://user-images.githubusercontent.com/18328279/200142920-6d4262e5-25e1-485e-a90f-41530b686dec.png)
- Obsidian v. 1.0.3
- Installed plugins:
  - Style Settings v. 0.4.12
  - Supercharged Links v. 0.9.3

**Comment by d2718nis** [comment]
There's a workaround: enable the CSS snippet by hand at `Options -> Appearance -> CSS snippets -> supercharged-links-gen`. This way it will stay enabled the next time Obsidian is restarted.

This issue most likely is related #137

**Comment by HEmile** [comment]
Wait, so you mean it gets disabled every time after reloading Obsidian? It's true that it's only enabled after you go to the supercharged links settings, but it should remain enabled after reloading Obsidian.

**Comment by d2718nis** [comment]
> Wait, so you mean it gets disabled every time after reloading Obsidian?

Yes, but only if it wasn't enabled by hand at the `Options -> Appearance -> CSS snippets -> supercharged-links-gen`

**Comment by HEmile** [comment]
Okay i guess that's a bug introduced by the latest obsidian versions
